react-native-libraries.json

```diff
@@ -10016,5 +10016,12 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/KroosX4V/react-native-orientation-manager",
+    "npmPkg": "react-native-orientation-manager",
+    "ios": true,
+    "android": true,
+    "windows": true
```

Comment:
For the out-of-tree platforms support (like Windows) we usually require an example app, since there were several false-positives compatibility marks in the past. 

To do not block this PR, we can ship it with mobile platforms, and update the Windows support when you will have the example available.